package.json

```diff
@@ -1,22 +1,19 @@
 {
-  "name": "react-training-RoboDex",
+  "name": "react-training-robodex",
```

Comment:
...actually, maybe we should change the name of this and the repo. it's not really intended to teach redux.